File: QTD.SPA/src/app/_DtoModels/Version_Procedure_Tool_Link/Version_Procedure_Tool_Link.ts
import { Entity } from '../Entity';
import { Version_Procedure } from '../Version_Procedure/Version_Procedure';
import { Version_Tool } from '../Version_Tool/Version_Tool';

export class Version_Procedure_Tool_Link extends Entity {
  version_ProcedureId!: any;
  version_ToolId!: any;
  version_Procedure!: Version_Procedure;
  version_Tool!: Version_Tool;
}

File: QTD2.Domain/Entities/Core/Version_Procedure_Tool_Link.cs
namespace QTD2.Domain.Entities.Core
{
    public class Version_Procedure_Tool_Link : Common.Entity
    {
        public int Version_ProcedureId { get; set; }

        public int Version_ToolId { get; set; }

        public virtual Version_Procedure Version_Procedure { get; set; }

        public virtual Version_Tool Version_Tool { get; set; }

        public Version_Procedure_Tool_Link()
        {
        }

        public Version_Procedure_Tool_Link(int version_ProcedureId, int version_ToolId)
        {
            Version_ProcedureId = version_ProcedureId;
            Version_ToolId = version_ToolId;
        }
    }
}
